modules/getting-started-openshift-gitops.adoc

```diff
@@ -0,0 +1,128 @@
+
+If you want to install your own instance for managing your cluster configurations or deploying your applications, you must deploy your own instance of ArgoCD. Using your own instance of ArgoCD helps you to continuously deliver your applications.
+
+.Installing ArgoCD
+
+To install an ArgoCD instance:
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Select *Operators* then select *Installed Operators*.
+. Create a new project or select an existing project, depending on where you want to install the ArgoCD instance.
+. Select *OpenShift GitOps* from the "Installed Operators" page.
+. Select the *ArgoCD* tab.
+. Create an ArgoCD instance and specify the following configurations:
+  * Name: <argocd-instance-name>
+  * Server > Route > Enable Route = true. This configuration creates an external OS route to access the ArgoCD server.
+
+.Deploying resources to a different namespace
+
+To grant ArgoCD the permissions to manage resources in multiple namespaces, create RoleBindings which provide ArgoCD admin permissions for that particular namespace. Use the example, OOTB ArgoCD instance, deployed in the openshift-gitops namespace.
+
+To create a RoleBinding:
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Go to *User Management* > *Role Bindings* > select *Create Binding*.
+  * You now see a form that guides you through the process of creating a RoleBinding:
+. For *Binding Type*, select *Namespace Role Binding (RoleBinding)*.
+  * If you only select *All Projects*, then you only need to provide the *Binding Type*.
+  * If the project is already selected as the one you want to create the RoleBinding for, the project does not ask for *Binding Type* because it assumes it is a RoleBinding.
+. Under *Role Binding*, provide a RoleBinding *Name* and *Namespace*.
+  * For the *Namespace*, specify the one you are granting permissions for.
+. For *Role*, select the *Cluster Role admin (CR admin)* from the dropdown list.
+. Under *Subject*, select *Service Account* and provide the *Subject Namespace* and *Name*. For example:
+  * Subject Namespace: openshift-gitops
+  * Subject Name: argocd-cluster-argocd-application-controller
+. To create the RoleBinding, select *Create*.
+
+.Configuring clusters
+
+After you deploy an ArgoCD instance, you can manage your cluster configurations:
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Update the existing *Subscription Object* for the GitOps operator.
+. Go the spec and add, *ARGOCD_CLUSTER_CONFIG_NAMESPACES*.
+. On the OpenShift Console, go to *Administration* > *CustomResourceDefinition* > *Subscription* > *Instances* then select *openshift-gitops-operator*.
```

Comment:
Remove steps on line 81, 82, and 83. Line 84 is where the steps actually start describing what is on line 81, 82 and 83 accurately.

 In the *Administrator* perspective of the web Console, navigate to...